include/RigidBody/Joints.h

```diff
@@ -163,6 +163,28 @@ class BIORBD_API Joints : public RigidBodyDynamics::Model
     int GetBodyRbdlIdToBiorbdId(
         const int idx) const;
 
+    ///
+    /// \brief Return the rbdl idx of child of each segments
+    /// \return the rbdl idx of child of each segments
+    ///
+    std::vector<std::vector<unsigned int> > GetChildRbdlIdx() const;
```

Comment:
I usually do not use "Get" in the name, but I am not against... I feel though that I would never capitalize the G (although some programmers do)

Reply:
Three functions now have a capital G, so I would need to modify these functions too, if you require uncapitalized g.

Reply:
`///
    /// \brief Return the biorbd body identification
    /// \param segmentName The name of the segment
    /// \return The biorbd body identification
    ///
    int GetBodyBiorbdId(
        const utils::String &segmentName) const;`

    `///
    /// \brief Return the rbdl body identification
    /// \param segmentName The name of the segment
    /// \return The rbdl body identification
    ///
    int GetBodyRbdlId(
        const utils::String &segmentName) const;`

    `///
    /// \brief Return the Biorbd body identification from rbdl
    /// \param idx The Rbdl body Id
    /// \return The Biorbd body identification
    ///
    int GetBodyRbdlIdToBiorbdId(
        const int idx) const;  `

Reply:
done